src/components/doc-view.js

```diff
@@ -2,8 +2,9 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import SectionDisplay from './section-display.js';
 import { documentShape, documentRevisionShape } from '../ui/default-prop-types.js';
+import { SECTION_ACTIONS } from '../ui/section-actions.js';
```

Comment:
Ha, that sometimes happens to mee too with the auto-import :laughing:

Reply:
😱 Na das kann nicht sein!